### ISSUE TITLE
[Amazon Linux] undefined symbol: gai_error

### ISSUE THREAD

**Issue by tillkruss** [cachewerk/relay#13]
It seems that AL2 builds are broken as of v0.4.0:

```
> php --ri relay

PHP Warning:  PHP Startup: Unable to load dynamic library 'relay.so' (tried: /usr/lib64/php/modules/relay.so (/usr/lib64/php/modules/relay.so: undefined symbol: gai_error), /usr/lib64/php/modules/relay.so.so (/usr/lib64/php/modules/relay.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0

Extension 'relay' not present.
```

```
docker build --pull --tag relay-amazon2 --file amzn2.Dockerfile .
docker run -it relay-amazon2 bash
php --ri relay
```

**Comment by tillkruss** [comment]
@michael-grunder I added docker build tests, looks like our CentOS7 is broken, that's what Amazon Linux 2 uses.

https://github.com/cachewerk/relay/runs/6816426459?check_suite_focus=true

**Comment by tillkruss** [comment]
This will be fixed in v0.4.1 👍
